src/js/controller/controller.js

```diff
@@ -262,6 +262,8 @@ define([
                 }
 
                 _checkAutoStart();
+
+                _model.set('preloaded', false);
```

Comment:
If this is a model default it should be set in model.js in the constructor. Is there an event listener on this that requires it be called here?